themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css

```diff
@@ -10191,7 +10196,7 @@ ul.social-links li a {
 }
 
 .title-bar__content {
-  border-color: #555 !important;
+  border-color: #a3a3a3 !important;
```

Comment:
@jacine should this be a variable ?

Reply:
nm, this is already a variable, looking at compiled file.

Reply:
Thanks Minnur! :)